src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -60,15 +84,68 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
         this.paramSubscription.unsubscribe();
     }
 
-    public groupExercises(selectedOrder: number): void {
+    get filterType() {
+        return ExerciseFilter;
+    }
+
+    /**
+     * Return the total number of exercises for the logged in user
+     */
+    getNrOfExercises(): number {
```

Comment:
Please type the full name of the method: `getNumberOfExercises`. It is more understandable that way.

Reply:
I now use the full name